### ISSUE TITLE
Add ROUGE metric

### ISSUE THREAD

**Issue by jshin1394** [google/metrax#27]

**Comment by priyanshscpp** [comment]
I’d like to work on this. Any guidance?

**Comment by jshin1394** [comment]
Hi Priyanshu, thank you so much for interest in this issue! I think you can get started by familiarizing yourself with the Keras definition of [RougeN](https://keras.io/api/keras_nlp/metrics/rouge_n/) and [RougeL](https://keras.io/api/keras_nlp/metrics/rouge_l/). You can also take a look at the source code in keras github.

**Comment by jshin1394** [comment]
@priyanshscpp Any update? If you don't want to work on this metric anymore, that is completely fine. Please let us know :)

**Comment by jshin1394** [comment]
Done
